src/components/notecontainer/notecontainer.jsx

```diff
@@ -43,19 +43,65 @@ export const NoteContainer = ({ title, text, color, date, time, id }) => {
   const removeArchiveNote = async () => {
     try {
       const response = await axios.post(
-        `/api/archives/restore/${id}`,
+        `/api/archives/restore/${_id}`,
         {},
```

Comment:
why are you giving an empty object?

Reply:
it's a way in mockbee to restore data from the API. We have to give an empty object in request body